scripts/test.sh

```
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+run_tests() {
+    local target_log=$1
+    local cargo_opts=$2
+
+    log "Running tests for $target_log..."
+    cargo test $cargo_opts --all-features
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)